spyder/plugins/appearance/confpage.py

```diff
@@ -147,6 +151,32 @@ def setup_page(self):
             tip=system_font_tip
         )
 
+        # Preview widgets
+        preview_editor_label = QLabel(_("Editor"))
+        self.preview_editor = SimpleCodeEditor(self)
+        self.preview_editor.setFixedWidth(260)
+        self.preview_editor.set_language('Python')
+        self.preview_editor.set_text(PREVIEW_TEXT)
+        self.preview_editor.set_blanks_enabled(False)
+        self.preview_editor.set_scrollpastend_enabled(False)
+
+        preview_interface_label = QLabel(_("Interface font"))
+        self.preview_interface = QLabel("Happy Spydering!")
```

Comment:
> I think it kind of make sense to allow users to edit the text since maybe they would like to check how other characters not visible in the preview default text could look. Also, since the preview for the monospace font can be edited, it also kind of makes sense for the other preview to be editable too, no? What do you think?

That's a good idea. I'll try to implement it with a `QPlainTextEdit` widget (right now I'm using a `QLabel`).

> Also, although I kind of understand trying to be appealing by using the Happy Spydering! text, maybe a more standard text like the selected font's name or using a translatable text as Interface Font Sample or Sample or even AaBbYyZz could make sense?

Good idea too. I'll change the text to `Sample text` to use a more standard one.

Reply:
> That's a good idea. I'll try to implement it with a `QPlainTextEdit` widget (right now I'm using a `QLabel`).

This turned to be not easy because it's not possible to vertically center text in `QPlainTextEdit` nor `QTextEdit` (see for instance this [SO question](https://stackoverflow.com/q/32941367/438386)) about it.

So, for now I'm going to make the editor preview read-only to make things consistent among the two previews.